### DISCUSSION TITLE
How to reset web password from ssh?

### DISCUSSION THREAD

**Discussion by 2600box** [mcguirepr89/BirdNET-Pi#1234] [Q&A]
Hello,

I seem to have improperly recorded the password I set for the web interface. I can access over ssh. Can I reset the web password from ssh?

Thanks

**Comment by rdz-oss** [comment]
It is in /etc/birdnet/birdnet.conf. . Edit e.g. via nano editor

sudo nano /etc/birdnet/birdnet.conf
